powsybl-parent/powsybl-parent-ws/pom.xml

```diff
@@ -25,17 +25,32 @@
     <description>Powsybl Parent for Web Services</description>
 
     <properties>
+    
+    	<java.version>11</java.version>
+    	
```

Comment:
update the readMe file 
a default target java version (8) => a default target java version (11)